modules/public/views.py

```diff
@@ -200,6 +200,11 @@ def snap_details(snap_name):
     except ApiError as api_error:
         status_code, error_info = _handle_errors(api_error)
 
+    # filter out banner and banner-icon images from screenshots
+    screenshot_urls = [
```

Comment:
Yep looks perfect. :+1:

Reply:
Actually thinking about it just a touch more, it might be nice to call it something other than `screenshot_urls`, just to make it clear that it's not *exactly* the same as the `screenshot_urls` set that comes from the API.